qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +89,8 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        wait_time = int(params.get("wait_time"))
+        time.sleep(wait_time)
```

Comment:
`hotplug_memory` using `simple_hotplug` and it will invoke `verify_hotplug` in the finish.
Is this step in order to wait for hpt to be active?